src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/WebJobs.Logging.ApplicationInsights.csproj

```diff
@@ -1,16 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>3.0.42$(VersionSuffix)</Version>
```

Comment:
Did not follow why are we removing 3.0.42? Is this value expected to come from somewhere else?